product_supplierinfo_for_customer/models/product_product.py

```diff
@@ -60,38 +60,27 @@ def _name_search(
         res.extend(self.browse(product_ids).name_get())
         return res
 
-    def _get_price_from_customerinfo(self, partner_id):
+    def _get_price_from_customerinfo(self, partner):
```

Comment:
It's still risky, as you are changing the signature of one method, and thus other extensions will fail in this. Please keep the same signature as previously. You can change the signature on a version migration.

Reply:
Done